README.md

```diff
@@ -29,6 +31,15 @@ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
 
 You can then run `helm search repo scalar-labs` to see the Scalar charts.
 
+### Pre-commit hook
+
+If you want to automatically generate `README.md` files and `values.schema.json` with a pre-commit hook.
+Then run:
+
+``` bash
+ln -s ../../.git-hooks/pre-commit .git/hooks/pre-commit
```

Comment:
Are you assuming you are somewhere specifically?

Reply:
It is assumed to be run as root on the repository.
To make it clear that you are running directly under the repository, make sure to move to root.

https://github.com/scalar-labs/helm-charts/pull/72/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R40